rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -521,7 +521,7 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
   RCUTILS_WARN_UNUSED
   rclcpp_lifecycle::LifecycleNode::OnSetParametersCallbackHandle::SharedPtr
   add_on_set_parameters_callback(
```

Comment:
I think it might be worth exposing the methods for adding/removing the new pre and post callbacks to the lifecycle node as well.

Reply:
yeah, make sense. Thanks!